Assets/_Project/Code/Models/Question.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using Newtonsoft.Json;
+
+public class Question { 
+
+    [JsonProperty("tresc")]
+    public string Tresc { get; private set; }
+    [JsonProperty("poprawneOdpowiedzi", Order = 2)]
+    private List<string> correctAnswers = new List<string>();
+    [JsonProperty("podpowiedzi", Order = 3)]
+    private List<string> answers = new List<string>();
+
+
+    public Question(string Tresc, List<string> correctAnswers, string odpA, string odpB, string odpC, string odpD)
+    {
+        this.Tresc = Tresc;
+        this.correctAnswers = correctAnswers; // podane jako lista poprawne warianty odpowiedzi
+
+
+        answers.Add(odpA);
+        answers.Add(odpB);
+        answers.Add(odpC);
+        answers.Add(odpD);
+    }
+
+    public bool IsCorrenct(string answer)
+    {
+        return correctAnswers.Contains(answer.Trim().ToLower());
+    }
+
+    public string Hint()
+    {
+        return $"A: {answers[0]}, B: {answers[1]}, C: {answers[2]}, D: {answers[3]}";
+    }
```

Comment:
To bardziej takie ToString(), poza tym i tak lepiej po prostu zrobić getter z answersami z AsReadOnly(), bo potem i tak trzeba tego stringa jakoś rozdzielić żeby w tabelce to pokazać